deploy/lvm-operator.yaml

```diff
@@ -1114,7 +1131,7 @@ spec:
                 values:
                 - openebs-lvm-controller
             topologyKey: "kubernetes.io/hostname"
-      priorityClassName: system-cluster-critical
+      priorityClassName: openebs-lvm-localpv-csi-controller-critical
```

Comment:
will it work for kube-system namespace? I remember there was some issue related to this.

Reply:
Yes

Reply:
I have tested locally once as well.